src/Protocol/ActivityPub/Receiver.php

```diff
@@ -1091,7 +1091,7 @@ private static function getReceivers(array $activity, string $actor, array $tags
 		if (!empty($actor)) {
 			$profile   = APContact::getByURL($actor);
 			$followers = $profile['followers'] ?? '';
-			$is_forum  = ($actor['type'] ?? '') == 'Group';
```

Comment:
`$actor` is a URL string coming from the function parameter, it can't be used to access a `type` key. On the other hand, this key is present in the return value of `APContact::getByURL`.